packages/rgbpp/README.md

```diff
@@ -49,4 +49,49 @@ const { txid: btcTxId } = await btcService.sendBtcTransaction(btcTx.toHex());
 
 // Post the BTC txId and ckbVirtualTxResult to the RGB++ Queue Service
 await btcService.sendRgbppCkbTransaction({ btc_txid: btcTxId, ckb_virtual_result: ckbVirtualTxResult });
-```
\ No newline at end of file
+```
+
+## Transfer all balance of an RGB++ Asset on BTC
+
+Similar to using the `buildRgbppTransferTx` function, the function `buildRgbppTransferAllTxs` will generate a list of RGB++ transaction groups (a transaction group includes a CKB virtual transaction and a BTC isomorphic transaction).
+
+You should sign all the PSBTs in the `transactions` and send all the BTC transactions, and then post all the BTC txIds and ckbVirtualTxResults to the RGB++ Queue Service. You can also review the transfer details in the `summary` object.
+
+```TypeScript
+const { transactions, summary } = await buildRgbppTransferAllTxs({
+  ckb: {
+    xudtTypeArgs,
+    collector,
+  },
+  btc: {
+    assetAddresses,
+    fromAddress,
+    toAddress,
+    dataSource,
+    feeRate,
+    pubkeyMap,
+  },
+  isMainnet,
+});
```

Comment:
### Discussion
Add the comments of https://github.com/ckb-cell/rgbpp-sdk/blob/b1865c2ac35ef3dec68afbd81e0736cbff80edd0/packages/rgbpp/src/rgbpp/types/xudt.ts#L45-L76 to this README, to provide a clear reference for developers working with `buildRgbppTransferAllTxs`, especially when dealing with `assetAddresses`, `fromAddress`, `toAddress` arguments.

Reply:
Updated at c7f37a3